hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileSystemDeleteHandler.java

```diff
@@ -35,21 +51,72 @@ public FileSystemDeleteHandler() {
         // Exists for injection
     }
 
+    /**
+     * This method is called during the pre-handle workflow.
+     *
+     * <p>Determines signatures needed for deleting a system file
+     *
+     * @param context the {@link PreHandleContext} which collects all information that will be
+     *     passed to {@code handle()}
+     * @throws NullPointerException if any of the arguments are {@code null}
+     */
     @Override
-    public void preHandle(@NonNull final PreHandleContext context) {
+    public void preHandle(@NonNull final PreHandleContext context) throws PreCheckException {
```

Comment:
This method does need to check that the entity ID is less than the configured max system entity ID, and fail otherwise.  The SystemFileDelete transaction is only valid for files with entity ID values in the correct range (0 < ID < MAX_SYSTEM_ENTITY).

Reply:
I fixed it but I need a property for the MAX_SYSTEM_ENTITY.

Reply:
Understood, I suspect you can add that in the appropriate config record (`LedgerConfig`?), but check with @hendrikebbers to be sure.